packages/core/database/src/index.ts

```diff
@@ -119,6 +124,15 @@ class Database {
     this.entityManager = createEntityManager(this);
   }
 
+  async repair(options: RepairOptions) {
+    // We only clean up components on startup
+    if (options.removeOrphanMorphTypes) {
```

Comment:
I would suggest to create a database repair service where we will implement different logic overtime instead of just have utils :)

Reply:
each function can be in it's own file but I would expose them from a single repair service to match with what is already done (schema, inspector, migration...)